### ISSUE TITLE
Search course by staff

### ISSUE THREAD

**Issue by ofiralexi** [TechnionYP5779/team1-bot#77]
Description : I want to get a name of lecturer and get all the courses he passed.
Given by the infromation of #75 :)

**Comment by ShaiZeevi** [comment]
@ofiralexi if you want to incorporate this feature into the current course filtering\ use a similar schema feel free to consult with me for whatever assistance you need

**Comment by ShaiZeevi** [comment]
Yes, I believe so too. What I mean is that if you want to use a similar technique feel free to consult
